migrations/2022-11-27-180447_custom_emojis/up.sql

```diff
@@ -0,0 +1,16 @@
+create table custom_emoji (
+  id serial primary key,
+  local_site_id int references local_site on update cascade on delete cascade not null,
+  shortcode varchar(128) not null,
```

Comment:
We should also add unique to some of these. The image_url, and shortcode should be unique.

Reply:
Yes, shortcode should be unique, and I'll add that in. Technically image_url doesn't need to be, but I suppose I'll add it too. Keyword shouldn't be unique because multiple emojis could have the same keyword, i.e. 'face' could pull up multiple emojis with a face.

Reply:
Fixed